libraries/AP_BattMonitor/AP_BattMonitor.h

```diff
@@ -51,18 +61,19 @@ class AP_BattMonitor
 
     // The BattMonitor_State structure is filled in by the backend driver
     struct BattMonitor_State {
-        cells       cell_voltages;      // battery cell voltages in millivolts, 10 cells matches the MAVLink spec
-        float       voltage;            // voltage in volts
-        float       current_amps;       // current in amperes
-        float       current_total_mah;  // total current draw since start-up
-        float       consumed_wh;        // total energy consumed in Wh since start-up
-        uint32_t    last_time_micros;   // time when voltage and current was last read
-        uint32_t    low_voltage_start_ms;  // time when voltage dropped below the minimum
-        float       temperature;        // battery temperature in celsius
-        uint32_t    temperature_time;   // timestamp of the last recieved temperature message
-        float       voltage_resting_estimate; // voltage with sag removed based on current and resistance estimate
-        float       resistance;         // resistance calculated by comparing resting voltage vs in flight voltage
-        bool        healthy;            // battery monitor is communicating correctly
+        cells       cell_voltages;             // battery cell voltages in millivolts, 10 cells matches the MAVLink spec
```

Comment:
Does the added value of indentation surpass the value of breaking git blame ?

Reply:
I've been caught! :)